#! /bin/sh /usr/share/dpatch/dpatch-run
## 03-override-jruby-home.dpatch by Sebastien Delafond <seb@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: jruby_home is at a specific location on Debian.

@DPATCH@
Index: jruby-1.5.0~rc1/bin/jruby
===================================================================
--- jruby-1.5.0~rc1.orig/bin/jruby.sh	2010-04-20 17:25:21.000000000 +0200
+++ jruby-1.5.0~rc1/bin/jruby.sh	2010-04-20 17:25:50.000000000 +0200
@@ -46,6 +46,7 @@
 else
   JRUBY_HOME=`dirname "$JRUBY_HOME_1"`  # the . dir
 fi
+JRUBY_HOME=/usr/lib/jruby/
 
 if [ -z "$JRUBY_OPTS" ] ; then
   JRUBY_OPTS=""
@@ -288,7 +289,7 @@
 JFFI_OPTS="-Djffi.boot.library.path=$JFFI_BOOT"
 
 if $cygwin; then
-  JRUBY_HOME=`cygpath --mixed "$JRUBY_HOME"`
+  JRUBY_HOME=/usr/lib/jruby/
   JRUBY_SHELL=`cygpath --mixed "$JRUBY_SHELL"`
   
   if [[ ( "${1:0:1}" = "/" ) && ( ( -f "$1" ) || ( -d "$1" )) ]]; then
